services/src/app.js

```diff
@@ -71,13 +74,19 @@ app.configure(channels);
 // Configure a middleware for 404s and the error handler
 app.use(express.notFound());
 
-/* Avoid cluttering the test logs with expected errors and exceptions */
-if (process.env.NODE_ENV != 'test') {
-  app.use(express.errorHandler({ logger }));
-}
-else {
-  app.use(express.errorHandler());
-}
+/* eslint-disable no-unused-vars */
+app.use((err, req, res, next) => { 
+  if (!err.statusCode) err.statusCode = err.code ? err.code : 500;
```

Comment:
It took me a bit of time to read this correctly, I think adding parenthesis would improve a bit readability (and also.

```js
err.statusCode = (err.code ? err.code : 500);
```

?

Reply:
Agree, fixing